build.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
```

Comment:
do we need to keep here the first few lines from test-manager:
https://github.com/ccxt/ccxt/blob/master/tests-manager.sh#LL3C3-LL10C10:~:text=bin/env%20bash-,if%20%5B%20%22%24%7BBASH_VERSION%3A0%3A1%7D%22%20%2Dlt,fi,-cache_path%3D%22